XenAdmin/XenAdmin.csproj

```diff
@@ -6871,7 +6932,9 @@
       <Install>true</Install>
     </BootstrapperPackage>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>

```

Comment:
I don't know how this got here, but let's remove it. Can do it in a separate PR.

Reply:
I added a commit to remove this.